Source/EasyNetQ/RabbitHutch.cs

```diff
@@ -146,7 +146,7 @@ public static IBus CreateBus(string connectionString, Action<IServiceRegister> r
                 VirtualHost = virtualHost,
                 UserName = username,
                 Password = password,
-                RequestedHeartbeat = requestedHeartbeat
+                RequestedHeartbeat = TimeSpan.FromSeconds(requestedHeartbeat)
```

Comment:
Maybe change `requestedHeartbeat` param to `TimeSpan` ?

Reply:
Done